src/components/studies/participants/ParticipantManagerPlaceholder.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react'
+import {makeStyles, Box} from '@material-ui/core'
+import RocketIcon from '../../../assets/participant-manager/participant_manager_rocket_icon.svg'
```

Comment:
we already have /participants directory in the assets. let's keep all the images there. Also since you're doing that work can you please move  the images our /ParticipantManagerPageSelector into the subdirectory of /participants named /paging (needs to be created) and update the references